setup/pkg/fieldlabs/policy.go

```diff
@@ -0,0 +1,29 @@
+package fieldlabs
+
+type Policy struct {
```

Comment:
Did we make these from scratch or copy? If copied can we point back to the original?

Reply:
Added the copy statements. Not everything is copied. Some are from scratch.